tensorflow_lite_support/ios/task/processor/sources/TFLDetectionResult+Helpers.h

```diff
@@ -35,11 +30,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (TFLDetectionResult *)detectionResultFromCDetectionResults:
     (TfLiteDetectionResult *)cDetectionResult;
```

Comment:
Do you want to change the param to const TfLiteDetectionResult?

Reply:
> Do you want to change the param to const TfLiteDetectionResult?

We should return a pointer as how it's implemented now because it aligns better with ObjC convention.

Reply:
const is not used for objects usually.

Reply:
@lu-wang-g FYI Google's ObjC implementation also doesn't use const pointer to pass object parameters. Here's an example from GTMHTTPFetcher, one of the most popular ObjC OSS project from Google.
https://github.com/google/gtm-http-fetcher/blob/a3e318b101410422bd510f9afd6bc9c28ec87019/Source/GTMHTTPFetchHistory.h#L134